oxidation/libparsec/crates/core_fs/src/storage/manifest_storage.rs

```diff
@@ -379,12 +396,22 @@ impl ManifestStorage {
         Ok((local_changes, remote_changes))
     }
 
+    pub(crate) fn get_cache_entry(&self, entry_id: EntryID) -> Arc<Mutex<CacheEntry>> {
```

Comment:
```suggestion
    pub(crate) fn get_or_insert_default_cache_entry(&self, entry_id: EntryID) -> Arc<Mutex<CacheEntry>> {
```
also add a doc-comment